src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -89,52 +93,12 @@ export class DotEditContentHtmlService {
         this.addContentContainer = identifier;
     }
 
-    private addContainerToolbar(): void {
+    private createContentToolBars(): void {
```

Comment:
The name doesn't match what this does, this method doesn't generate the toolbar it adds it. It should so addContainerToolbar makes more sense.